migrations.lock

```diff
@@ -1094,13 +1096,7 @@ translations
  0008_remove_couch_id
  0009_auto_20200924_1753
 two_factor
- 0001_initial
- 0002_auto_20150110_0810
- 0003_auto_20150817_1733
- 0004_auto_20160205_1827
- 0005_auto_20160224_0450
- 0006_phonedevice_key_default
- 0007_auto_20201201_1019
+ (no migrations)
```

Comment:
This is a bit surprising. Do you know why these went away? Were they squashed and moved to the `phonenumber` app?

Reply:
Yeah my understanding is they were squashed and moved to the `phonenumber` app to prevent new projects from needing to include the now optional `phonenumber_field` dependency when using this library.